### PR TITLE
[WFCORE-820] Attribute for-hosts for Security Realm with Kerberos doe…

### PR DESCRIPTION
**Pull request by TomasHofman**
…s not select keytab in defined order

https://issues.jboss.org/browse/WFCORE-820
https://bugzilla.redhat.com/show_bug.cgi?id=1228228

### CODE REVIEW

**Comment by TomasHofman** [comment]
@darranl Would you review please?

File: domain-management/src/test/java/org/jboss/as/domain/management/security/KeytabIdentityFactoryServiceTestCase.java [hunk 1]
```diff
@@ -0,0 +1,174 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2012, Red Hat Middleware LLC, and individual contributors
```
Comment (darranl) [inline domain-management/src/test/java/org/jboss/as/domain/management/security/KeytabIdentityFactoryServiceTestCase.java]: It is 2015

**Comment by darranl** [comment]
The actual code changes look fine - however IMO when it comes to backwards compatibility unless the current behaviour is so broken it is impossible for it to actually be used by anyone the current behaviour has to take priority over documentation.

In this case I think it will be better to just update the documentation.

Should the issue described here actually occur users do have the opportunity to add an additional for-hosts value to restore the desired behaviour.

**Comment by bstansberry** [comment]
Given the comments above I'm closing this.

**Comment by TomasHofman** [comment]
I opened new PR with modified documentation: https://github.com/wildfly/wildfly-core/pull/887

**Pull request closed without merging**